src/language/rule/rule.ts

```diff
@@ -138,6 +138,10 @@ export class Replacement {
         return replacements.reduce((text, r) => r.apply(text), content);
     }
 
+    public static replaceNode(node: ts.Node, text: string): Replacement {
```

Comment:
please add an optional `sourceFile` parameter, that will be passed to `node.getStart` (for performance)